style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+    font-family: "MavenPro";
+    src: url(./assets/fonts/MavenPro-Regular.woff) format("woff");
+    font-style: normal;
+    font-weight: 400;
+}
+
+@font-face {
+    font-family: "MavenPro";
+    src: url(./assets/fonts/MavenPro-Bold.woff) format("woff");
+    font-style: bold;
+    font-weight: 700;
+}
+
+@font-face {
+    font-family: "Lato";
+    src: url(./assets/fonts/Lato-Regular.woff) format("woff");
+    font-style: normal;
+    font-weight: 400;
+}
+
+@font-face {
+    font-family: "Lato";
+    src: url(./assets/fonts/Lato-Bold.woff) format("woff");
+    font-style: bold;
+    font-weight: 700;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    margin: 0;
+    font-weight: 700;
+    font-family: 'MavenPro', sans-serif;
+    color: #454545;
+}
+
+img {
+    display: block;
+    width: 100%;
+    object-fit: cover;
+    height: 100%;
+}
+
+body {
+    max-width: 1024px;
+    margin: 0 auto;
+    padding-left: 20px;
+    font-family: 'Lato', sans-serif;
```

Comment:
would be great to add at least one fallback for font 
https://webref.ru/css/font-family